src/Behat/Symfony2Extension/ServiceContainer/Symfony2Extension.php

```diff
@@ -72,8 +72,8 @@ public function configure(ArrayNodeDefinition $builder)
                     ->addDefaultsIfNotSet()
                     ->children()
                         ->scalarNode('bootstrap')->defaultValue(self::DEFAULT_KERNEL_BOOTSTRAP)->end()
-                        ->scalarNode('path')->defaultValue('app/AppKernel.php')->end()
-                        ->scalarNode('class')->defaultValue('AppKernel')->end()
+                        ->scalarNode('path')->defaultValue(class_exists('App\Kernel') ? 'src/AppKernel.php' : 'app/AppKernel.php')->end()
```

Comment:
this path should default to null if the class exists, as it means it can be autoloaded (and so does not need to require any file) before instantiating